python/pyspark/sql/connect/functions/builtin.py

```diff
@@ -4229,6 +4229,90 @@ def hll_union(
 hll_union.__doc__ = pysparkfuncs.hll_union.__doc__
 
 
+def theta_sketch_agg(
+    col: "ColumnOrName",
+    lgNomEntries: Optional[Union[int, Column]] = None,
+) -> Column:
+    if lgNomEntries is None:
+        return _invoke_function_over_columns("theta_sketch_agg", col)
+    else:
+        return _invoke_function_over_columns("theta_sketch_agg", col, lit(lgNomEntries))
```

Comment:
super nit, this string is repeated from L4237 above, can we dedup it? Same for other new code below, and also L4190 and L4203 above.